client/src/components/Dashboard/CreateNewRelay/CreateNewRelay.js

```diff
@@ -1,5 +1,85 @@
 import React from 'react';
+import {
+  Form, Input, Row, Col, Button, Container, Label,
+} from 'reactstrap';
+import Header from '../../common';
+import ArrowRight from '../../../assets/icons/angle-right-solid.svg';
+import ArrowDown from '../../../assets/icons/angle-down-solid.svg';
 
-const CreateNewRelay = () => <h1>CreateNewRelay</h1>;
+const mobileScreen = window.innerWidth <= 425;
+
+const CreateNewRelay = () => {
+  return (
+    <>
+      <Header />
+      <Container>
+        <Form>
+          <Row>
+            <Col className="align-center">
+              <h3 className="heading mt-5 mb-5"> Create New Relay</h3>
+            </Col>
+          </Row>
+          <Row>
+            <Col sm="12" md="4">
+              <Label>Select App 1</Label>
+              <Input type="select" name="app1" id="appSelect1" required>
+                <option>1</option>
+                <option>2</option>
+                <option>3</option>
+                <option>4</option>
+                <option>5</option>
+              </Input>
+            </Col>
+            <Col className="align-center mt-4">
+              {(mobileScreen)
+                ? <img src={ArrowDown} alt="arrow" width="30px" />
+                : <img src={ArrowRight} alt="arrow" width="30px" />}
+            </Col>
+            <Col sm="12" md="4">
+              <Label>Select App 2</Label>
+              <Input type="select" name="app2" id="appSelect2" required>
```

Comment:
You can create a generic select component which will take name and options as props.

Reply:
Done